README.md

```diff
@@ -84,28 +84,32 @@ the workshop coordinators will want the former URL, not the latter.)
 ## Manual Setup
 
 You can set up your repository manually instead of using the automated
-`create` script.  As above, we will assume that your user ID is
-`ghopper` and the identifier for your workshop is `2015-07-01-esu`.
+`create` script.  Where you see <YOURUSERNAME> replace it with your 
+github userID. The identifier for your workshop is the start date and 
+the location of the workshop <YEAR-MONTH-DAY-LOCATION> e.g. 
```

Comment:
Replace line with

~~~
the location of the workshop `YEAR-MONTH-DAY-LOCATION` e.g.
~~~